proto/src/main/protobuf/FlightsMessage.proto

```diff
@@ -74,9 +74,10 @@ message FeedStatusMessage {
 }
 
 message TotalPaxSourceMessage {
-    optional int32 pax = 1;
     optional string feedSource = 2;
     optional PassengersMessage passengers = 3;
+
+    optional int32 pax__OLD = 1;
```

Comment:
I've just realised you also need to handle deserialising existing data where we've got `pax_OLD`